app/data/models/returns.js

```diff
@@ -0,0 +1,37 @@
+module.exports = (sequelize, DataTypes) => {
+  const returns = sequelize.define('returns', {
+    returnsId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+    schemeId: DataTypes.INTEGER,
+    sourceSystem: DataTypes.STRING,
+    paymentId: DataTypes.STRING,
+    paymentJobNumber: DataTypes.STRING,
+    fesCode: DataTypes.STRING,
+    traderNumber: DataTypes.STRING,
+    transactionNumber: DataTypes.STRING,
+    invoiceNumber: DataTypes.STRING,
+    sbi: DataTypes.INTEGER,
+    frn: DataTypes.BIGINT,
+    agreementNumber: DataTypes.STRING,
+    claimNumber: DataTypes.STRING,
+    currency: DataTypes.STRING,
+    value: DataTypes.INTEGER,
+    settlementDate: DataTypes.STRING,
+    paymentType: DataTypes.STRING,
+    reference: DataTypes.STRING,
+    referenceId: DataTypes.STRING,
+    bankAccount: DataTypes.STRING,
+    batchNumber: DataTypes.STRING,
+    settled: DataTypes.BOOLEAN,
+    valueEUR: DataTypes.INTEGER,
+    exchangeRate: DataTypes.STRING,
+    detail: DataTypes.STRING,
+    ledger: DataTypes.STRING,
+    filename: DataTypes.STRING
+  },
+  {
+    tableName: 'returns',
+    freezeTableName: true,
+    timestamps: false
+  })
+  return returns
```

Comment:
As we're not adding any associations you could just return the definition immediately before putting it in a variable first.

Reply:
I'm not sure if this will be the case, but what if there are associations in future? What's the likelihood of that happening?

Reply:
There might be in the future but we'd need to change this anyway if there were.

at the moment we've essentially got

```
function () {
   const model = createModel()
  return model
}
```

When we could just have

```
function () {
  return createModel()
}
```